hexrd/xrdutil/__init__.py

```diff
@@ -0,0 +1,7 @@
+from .utils import *
+
+# For now import these private members!
+from .utils import _project_on_detector_plane
+from .utils import _fetch_hkls_from_planedata
+from .utils import _filter_hkls_eta_ome
+from .utils import _fetch_hkls_from_planedata
```

Comment:
Looks like `_fetch_hkls_from_planedata` is imported twice!

Reply:
Fixed up